package.json

```
@@ -15,7 +16,8 @@
     "prepare": "husky install"
   },
   "dependencies": {
-    "next": "^13.2.4",
+    "next": "^13.4.4",
+    "next-sitemap": "^4.1.3",
```

```suggestion
    "next-sitemap": "^4.1.3",
```
For the generation of sitemap manually have their pros and cons, the pluses of the sitemap output is dynamic and always contains current and updated data, no problem to generate it manually based on [this example](https://vercel.com/guides/how-do-i-generate-a-sitemap-for-my-nextjs-app-on-vercel), but unfortunately this solution is not applicable to all, there may be difficulties with obtaining all paths, whether it is headless cms or static md files in the project, so I would use the `next-sitemap` plugin that closes these issues although it has a minus in that the generation of the file occurs only when building